husky_components/search/main.js

```diff
@@ -22,8 +21,8 @@ define([], function() {
 
     var templates = {
             skeleton: [
-                '<a class="fa-search fa-flip-horizontal search-icon" href="#"></a>',
-                '<a class="fa-times-circle remove-icon" href="#"></a>',
+                '<span class="fa-search fa-flip-horizontal search-icon"></span>',
+                '<span class="fa-times-circle remove-icon"></span>',
```

Comment:
It might make it easier to maintain, but it will definitely be less usable. It's impossible to use screenreaders or similar tools like that. And shouldn't a `onclick="return false;"` in the html be enough?